bin/run_demo.sh

```diff
@@ -109,18 +109,21 @@ refresh_exe
 # Run the init step
 pipetask --long-log run -b "$tmp_butler" -i "$incoll" --output-run "$exerun" --init-only --register-dataset-types --qgraph "$graph_file" --extend-run
 
+NODES=( $(pipetask qgraph -b "$tmp_butler" -g "$graph_file" --show-qgraph-header \
+    |python -c 'import json,sys;nodes=[id for id, _ in json.load(sys.stdin)["Nodes"]];print(" ".join(nodes))') )
```

Comment:
Consider using `jq` here for extracting the relevant content from the JSON. It's optimized for that.

Reply:
That would have been my first choice, but jq is not on every system, so I went with something that was

Reply:
it's part of rubin-env so should be fine but I'll defer to @ktlim in case he has a preference.

Reply:
Definitely suitable for `pipelines_check`.  And it's a pretty minimal dependency (single binary, no runtime dependencies) for any non-Rubin use of G3MW.